source/reference/minio-server/settings/metrics-and-logging.rst

```diff
@@ -0,0 +1,983 @@
+.. _minio-server-envvar-metrics-logging:
+
+============================
+Metrics and Logging Settings
+============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+This page covers settings that control behavior related to MinIO metrics and logging. 
+See :ref:`minio-metrics-and-alerts` for more information.
+
+These settings configure publishing regular :mc:`minio server` logs and audit logs to an HTTP webhook. 
+See :ref:`minio-logging` for more complete documentation.
+
+- :ref:`minio-server-envvar-logging-regular`
+- :ref:`minio-server-envvar-logging-audit`
+- :ref:`minio-server-envvar-logging-audit-kafka`
+
+Prometheus Authentication
+-------------------------
+
+This setting controls how MinIO authenticates to Prometheus.
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :selected:
+
+      .. envvar:: MINIO_PROMETHEUS_AUTH_TYPE
+
+   .. tab-item:: Configuration Setting
+
+      There is not a configuration setting option.
```

Comment:
```suggestion
      This setting does not have a configuration variable setting.
```

Reply:
Seems there a couple variants for the _nothing to see here_ text. I'm not sure which you prefer, I'll stop flagging them for the moment.

Reply:
I'll add an include for that to make sure they are consistent (and easier to change later, if need be). Then will update throughout.